src/catkin_pkg/changelog_generator.py

```diff
@@ -218,14 +218,17 @@ def replace_section(match):
     return data if count == 1 else None
 
 
+def version_from_tag(tag_name):
+    return tag_name.replace('v', '') if tag_name is not None else None
```

Comment:
Please modify this logic to only match the specific pattern accepted elsewhere: the `v` needs to be the first character and shouldn't be anywhere else in the string.

Reply:
@flixr Friendly ping.

Reply:
done